flink-rpc/flink-rpc-core/src/main/java/org/apache/flink/runtime/rpc/RpcUtils.java

```diff
@@ -41,9 +41,10 @@
 public class RpcUtils {
 
     /**
-     * <b>HACK:</b> Set to 21474835 seconds, Akka's maximum delay (Akka 2.4.20). The value cannot be
-     * higher or an {@link IllegalArgumentException} will be thrown during an RPC. Check the private
-     * method {@code checkMaxDelay()} in {@link akka.actor.LightArrayRevolverScheduler}.
+     * <b>HACK:</b> Set to 21474835 seconds, Pekko's maximum delay (Akka 2.4.20). The value cannot
```

Comment:
```suggestion
     * <b>HACK:</b> Set to 21474835 seconds, Pekko's maximum delay (Pekko 1.0.0). The value cannot
```

Reply:
I dont know if that is the maximum delay. I'd keep it as the source of the original value.